src/routes/api/user.route.js

```diff
@@ -1,11 +1,18 @@
 const express = require('express');
 
 const { validate } = require('../../middlewares');
+const { auth, author } = require('../../middlewares/auth.middleware');
```

Comment:
đoạn này import chung từ middleware
```suggestion
const { auth, author, validate  } = require('../../middlewares');
```